dll/win32/imm32/ime.c

```diff
@@ -1791,8 +1791,8 @@ ImmGetImeMenuItemsA(HIMC hIMC, DWORD dwFlags, DWORD dwType,
 {
     TRACE("(%p, 0x%lX, 0x%lX, %p, %p, 0x%lX)\n",
           hIMC, dwFlags, dwType, lpImeParentMenu, lpImeMenu, dwSize);
-    return Imm32GetImeMenuItemsAW(hIMC, dwFlags, dwType, lpImeParentMenu, lpImeMenu,
-                                  dwSize, TRUE);
+    return ImmGetImeMenuItemsAW(hIMC, dwFlags, dwType, lpImeParentMenu, lpImeMenu,
+                                dwSize, TRUE);
```

Comment:
You did tweak the spaces indentation here after the line-break, but you didn't do the same in the case of line 362. Also not in compstr.c.

(Maybe the most sustainable approach to avoid future whitespace-grooming would be eliminating that line-break altogether, and accept having a little longer lines?)

Reply:
OK. I used longer lines.